clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/KafkaConsumerServiceDelegator.java

```diff
@@ -71,31 +71,18 @@ public KafkaConsumerServiceDelegator(
     this.isAAWCStoreFunc = vt -> storeVersionAAWCFlagMap.computeIfAbsent(vt, ignored -> isAAWCStoreFunc.apply(vt));
     ConsumerPoolStrategyType consumerPoolStrategyType = serverConfig.getConsumerPoolStrategyType();
 
-    // For backward compatibility, if the dedicated consumer pool for AA/WC leader is enabled, we will use it.
-    // TODO: Remove this boolean check after new pooling strategy is verified in production environment.
-    if (serverConfig.isDedicatedConsumerPoolForAAWCLeaderEnabled()) {
-      this.consumerPoolStrategy = new AAOrWCLeaderConsumerPoolStrategy();
```

Comment:
If we want to achieve the goal of removing AAOrWCLeaderConsumerPoolStrategy, it is better to also delete its implementation in this PR